plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/rest/TrinoIcebergRestCatalogFactory.java

```diff
@@ -85,6 +87,7 @@ public synchronized TrinoCatalog create(ConnectorIdentity identity)
             ImmutableMap.Builder<String, String> properties = ImmutableMap.builder();
             properties.put(CatalogProperties.URI, serverUri.toString());
             warehouse.ifPresent(location -> properties.put(CatalogProperties.WAREHOUSE_LOCATION, location));
+            prefix.ifPresent(prefix -> properties.put("prefix", prefix));
```

Comment:
Can this be added to `CatalogProperties` in Iceberg?

Reply:
This is specific to single catalog (not all catalogs). Hence, it was not added I guess. 

Plus change at Iceberg needs a release, which will take time. We can update in the followup if the community agrees on it.